### DISCUSSION TITLE
Copyright

### DISCUSSION THREAD

**Discussion by karmstrong909** [community/community#136279] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am helping develop a website and am writing an article on Fractals as found in Nature. May I use the images in your tutorials if I acknowledge the source of them, and provide a link for readers to contact and download Xaos. If so, how would you like the credit to read?
Many thanks Keith Armstrong

**Comment by ebndev** [comment]
Hi @karmstrong909,

Thanks for being a part of the GitHub Community, we're glad you're here!

It looks like you're looking for support from `Xaos`. You might want to try asking for help somewhere that focuses on this project, such as opening an issue in the project's repository or seeing if they have a support page.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!

**Comment by karmstrong909** [comment]
Hi,
I asked about copyright permission on  using images in tutorials.
I did not see any comments on this. As I am new to OPen Source does that
mean the images contained or generated are completely free of copyright adn
may be used without seeking further permissions?
I will of course credit the program wherefrom the images have been drawn.
Best wishes
Keith

**Reply by ebndev** [reply]
It looks like they have a GPL license in the repo which does allow you to use images that are hosted in there. 

However, regarding how to credit them or any additional questions, I suggested opening an issue in [their repository](https://github.com/xaos-project/XaoS/issues) as this is a public forum and the owners of the repo won't see this discussion :)

**Comment by karmstrong909** [comment]
Thanks.
Best wishes
Keith
